__tests__/app.test.js

```diff
@@ -83,14 +82,85 @@ describe("4. GET /api/reviews/:review_id", () => {
       .then((response) => {
         expect(response.body.msg).toBe("Invalid review ID provided");
       });
-  })
-test("When user submits a valid request but there is not a matching review ID, function to notify user",()=>{
-  
-  return request(app)
-  .get("/api/reviews/100")
-  .expect(200)
-  .then((response) => {
-    expect(response.body.msg).toBe("No matching review id");
+  });
+  test("When user submits a valid request but there is not a matching review ID, function to notify user", () => {
+    return request(app)
+      .get("/api/reviews/100")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.msg).toBe("No matching review id");
+      });
+  });
+});
+
+describe("6. GET /api/reviews/:review_id/comments", () => {
+  test("Function gives requested information when given the correct ID", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        const result = [
+          {
+            comment_id: 5,
+            votes: 13,
+            created_at: "2021-01-18T10:24:05.410Z",
+            author: "mallionaire",
+            body: "Now this is a story all about how, board games turned my life upside down",
+            review_id: 2,
+          },
+          {
+            comment_id: 1,
+            votes: 16,
+            created_at: "2017-11-22T12:43:33.389Z",
+            author: "bainesface",
+            body: "I loved this game too!",
+            review_id: 2,
+          },
+          {
+            comment_id: 4,
+            votes: 16,
+            created_at: "2017-11-22T12:36:03.389Z",
+            author: "bainesface",
+            body: "EPIC board game!",
+            review_id: 2,
+          },
+        ];
+        expect(response.body.reviewComments).toEqual(result);
+      });
+  });
+  test("The Recieved data is to have the most recent comments first", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.reviewComments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("When client inputs an ID that is not a number, the Function will throw an error", () => {
+    return request(app)
+      .get("/api/reviews/;Drop table/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid review ID provided");
+      });
+  });
+  test("When Review id is valid but there is no matching data, Function to respond appropriately ", () => {
```

Comment:
This should be more semantic, for example: Status: 200 responds with an empty array when a valid ID is passed and there a no comments.